secretflow/ml/nn/sl/sl_model.py

```diff
@@ -292,6 +299,7 @@ def fit(
             self._workers[self.device_y].reset_metrics()
             self._workers[self.device_y].on_epoch_begin(epoch)
             fuse_net_num_returns = sum(self.basenet_output_num.values())
+            hiddens_buf = [None] * (self.pipeline_size - 1)
```

Comment:
`hidden_buf` will not be consumed completely, this is a bug?

Reply:
resolved